### PR TITLE
Fixed setup cmd on toolbox

### PR DESCRIPTION
**Pull request by khannaabhi**
1)changed mounting path to support toolbox
2)fixed label, exposed port of gateway in upgrade
3)even added restarting containers of sc, which was to be added later
4)created functions for mount path
5)tested setup on both toolbox and docker

### CODE REVIEW

File: space-cli/cmd/utils/paths.go [hunk 1]
```diff
@@ -50,3 +51,51 @@ func getHomeDirectory() string {
...
+	}
+	return GetSecretsDir()
+}
+
+// GetMountTempSecretsDir returns the path of the temp secret dir to be mounted in space cloud
+func GetMountTempSecretsDir() string {
+	if runtime.GOOS == "windows" {
+		return fmt.Sprintf("%s/.space-cloud/secrets/temp-secrets", getHomeDirectoryToolBox())
+	}
+	return GetSecretsDir()
```
Comment (sharadregoti) [inline space-cli/cmd/utils/paths.go:84]: I believe there should be a function call to GetTempSecretsDir( )

File: space-cli/cmd/utils/paths.go [hunk 2]
```diff
@@ -50,3 +51,51 @@ func getHomeDirectory() string {
...
+	if runtime.GOOS == "windows" {
+		return fmt.Sprintf("%s/.space-cloud/routing-config.json", getHomeDirectoryToolBox())
+	}
+	return GetSpaceCloudRoutingConfigPath()
+}
+
+// getHomeDirectoryToolBox gets home directory to support setup on window
+func getHomeDirectoryToolBox() string {
+	homeDrive := strings.ToLower(strings.Split(os.Getenv("HOMEDRIVE"), ":")[0])
+	homePath := strings.ReplaceAll(os.Getenv("HOMEPATH"), "\\", "/")
```
Comment (sharadregoti) [inline space-cli/cmd/utils/paths.go:98]: Write in a comment what is the value of HOMEDRIVE & HOMEPATH 
// HOMEDRIVE : "\\path\\"

File: space-cli/cmd/utils/paths.go [hunk 3]
```diff
@@ -50,3 +51,51 @@ func getHomeDirectory() string {
...
+// GetMountRoutingConfigPath returns the path of the routing config to be mounted in space cloud
+func GetMountRoutingConfigPath() string {
+	if runtime.GOOS == "windows" {
+		return fmt.Sprintf("%s/.space-cloud/routing-config.json", getHomeDirectoryToolBox())
+	}
+	return GetSpaceCloudRoutingConfigPath()
+}
+
+// getHomeDirectoryToolBox gets home directory to support setup on window
+func getHomeDirectoryToolBox() string {
```
Comment (sharadregoti) [inline space-cli/cmd/utils/paths.go:96]:
```suggestion
func getWindowsUserDirectory() string {
```

File: space-cli/cmd/utils/paths.go [hunk 4]
```diff
@@ -50,3 +51,51 @@ func getHomeDirectory() string {
...
 	return fmt.Sprintf("%s/accounts.yaml", GetSpaceCloudDirectory())
 }
+
+// GetMountHostsFilePath returns the path of the hosts files to be mounted in in space cloud
+func GetMountHostsFilePath() string {
+	if runtime.GOOS == "windows" {
+		return fmt.Sprintf("%s/.space-cloud/hosts", getHomeDirectoryToolBox())
+	}
+	return GetSpaceCloudHostsFilePath()
+}
```
Comment (sharadregoti) [inline space-cli/cmd/utils/paths.go:61]: do we need to create another function?  just do the check for windows in GetSpaceCloudHostsFilePath() itself. Do this for all the function.

**Comment by YourTechBud** [comment]
@sharadregoti and @khannaabhi  link the issues associated with this PR

**Comment by sharadregoti** [comment]
fixes #1076 #1074

**Pull request merged**